packages/wrangler/src/user/auth-variables.ts

```diff
@@ -111,3 +111,7 @@ export const getCloudflareAccessToken = async () => {
 
 	return getAccessToken(getAuthDomainFromEnv());
 };
+
+export const getR2SqlAPITokenFromEnv = getEnvironmentVariableFactory({
+	variableName: "CLOUDFLARE_R2_SQL_TOKEN",
+});
```

Comment:
This doesn't need an additional token. Instead, what about requiring users to provide `CLOUDFLARE_API_TOKEN`, which is our existing documented mechanism for accepting tokens in Wrangler?